data/schema/core/actionwml.cfg

```diff
@@ -466,9 +466,12 @@
 		max=infinite
 		{INSERT_TAG}
 		{SIMPLE_KEY map map_data}
+		{SIMPLE_KEY map_data map_data}
 		{SIMPLE_KEY map_file string}
 		{DEFAULT_KEY expand s_bool no}
 		{DEFAULT_KEY shrink s_bool no}
+		{SIMPLE_KEY x s_coordinates}
+		{SIMPLE_KEY y s_coordinates}
```

Comment:
Well, I also added some documentation for them to the wiki, since they do work.

Reply:
My point was that they _won't_ work outside of LoW, because the functionality is implemented [here in LoW's code](https://github.com/wesnoth/wesnoth/blob/master/data/campaigns/Legend_of_Wesmere/lua/wml_tags.lua#L31). Unless of course someone also added them to the C++, but I'm pretty sure that's not the case.

I think the proper quick-fix for this would be to rename LoW's overridden tag so that it doesn't produce schema errors. The proper long-term fix would probably be adding new tags such as `[shift_map]` or `[resize_map]` that do what LoW wants.

Reply:
Ah.  Well, that's annoying.  I've renamed it in LoW to [replace_map_section], in that case.